ext/dom/characterdata.c

```diff
@@ -364,12 +364,12 @@ PHP_METHOD(DOMCharacterData, remove)
 
 PHP_METHOD(DOMCharacterData, after)
 {
-	int argc;
+	int argc = 0;
```

Comment:
Those all should be  ``uint32_t`` in theory, but that's more a ``master`` branch thing.
```suggestion
	uint32_t argc = 0;
```

Reply:
Actually that's already done in master :D